src/test/resources/pages/NJIfeedbackSessionRemind.html

```diff
@@ -465,7 +465,7 @@ <h5 class="col-sm-2">
                         Open
                     </span>
                 </td>
-                <td class="session-response-for-test recent">0 / 9</td>
+                <td class="session-response-for-test recent">{*}</td>
```

Comment:
It is, but this sometimes cause the test to fail because the unpredictable ajax loading speed of the response rate. Since the testing purpose in this action is to verify the overall functionality of instructor and student pages for newly joint instructor. Changing it here will make this "long" test avoid being failed by irrelevant issues.